packages/devtools-core/src/index.tsx

```diff
@@ -24,6 +24,8 @@ const DevToolsApp = (props: Props) => (
     >
       {props.children}
     </App>
+    {/* Temporary fix for react-modal (at least in kanban example) */}
+    <div id="modal" className="modal" />
```

Comment:
Why does this modal have to live outside of the iframe?

Reply:
If you mean to move it into the iframe/UserAppContainer inside devtools, happy to do that.
If this is a more general question, it's because react-modal was throwing an error about there being no element with "#modal" (which was defined as needed by Modal.setAppElement("#modal") in the kanban example). This was a quick fix to get rid of that, but will need some generalizing

Reply:
Hmm, not sure why the div with id modal was not in the dom. Anyway, I think this fine for now. Maybe make an issue so we remember to address this later.